library/src/scripts/components/LinkAsButton.tsx

```diff
@@ -29,14 +30,14 @@ export default class LinkAsButton extends React.Component<IProps> {
     };
 
     public render() {
-        const { baseClass, className, title, ariaLabel, to, children, ...restProps } = this.props;
+        const { baseClass, className, title, ariaLabel, to, children, tabIndex, ...restProps } = this.props;
         const componentClasses = classNames(baseClass, className);
         return (
             <SmartLink
                 className={componentClasses}
                 title={title}
                 aria-label={ariaLabel || title}
-                tabIndex={-1}
+                tabIndex={tabIndex ? tabIndex : 1}
```

Comment:
To not have it hardcoded back and fourth, probably it is better to use some kind of object property or constant?